native_client/ctcdecode/Makefile

```diff
@@ -23,15 +23,22 @@ clean-keep-common:
 
 clean: clean-keep-common
 	rm -f common.a
+	rm workspace_status.cc
+	rm -fr bazel-out/
 
-bindings: clean-keep-common
+workspace_status.cc: ../workspace_status.h
+	mkdir -p bazel-out/ && \
+		../bazel_workspace_status_cmd.sh > bazel-out/stable-status.txt && \
+		../gen_workspace_status.sh > $@
```

Comment:
I really don't like replicating this little subset of Bazel logic here for this, but I can't think of a better way to do it :(

Reply:
I agree, but I also don't get a better idea.